docs/pages/feature-matrix.mdx

```diff
@@ -112,6 +112,8 @@ certificates, access control, and auditability.
 |**Integration:** Supports open-source policy agents, dev tool APIs, and Cloud IAM. Others include Jenkins, Github actions, Terraform Cloud, AWS Roles anywhere and more. | ✔ | ✔ | ✔ |
 |**HSM and TPM support** for bootstrapping, joining, and encryption | ✔ | ✔ | ✖ |
 |**Open Standards** \- JWT, SPIFFE, x509 and others to avoid vendor lock-in | ✔ | ✔ | ✔ |
+|**External PKI integration:** Configure an external PKI hierarchy to use for issuing SPIFFE SVIDs | ✔ | ✔ | ✖ |
+|**Sigstore Attestation:** Enforce validation of container supply-chain security when issuing SPIFFE SVIDs  | ✔ | ✔ | ✖ |
```

Comment:
Since it seems like we're using sentence case in the first column, I'm just confirming whether "Sigstore Attestation" is a product name (or otherwise a proper noun), or if "attestation" is a general computing term and needs to be lowercased.

Reply:
[6b9a04d](https://github.com/gravitational/teleport/pull/54561/commits/6b9a04db901f4dd983c5fa89215b55d692c251a7)